windows/src/engine/keyman/UfrmKeymanMenu.pas

```diff
@@ -726,18 +744,6 @@ procedure TfrmKeymanMenu.TntFormKeyDown(Sender: TObject; var Key: Word;
   if not Assigned(Fmnu) or not Assigned(Fmnu.Items) then Exit;  // I2884
 
   case Key of
-    {VK_NEXT:
```

Comment:
Using the arrow keys already scrolls the list, so no need to overload pgup/dn for this.